src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/service/ConfigurationAnchorServiceImpl.java

```diff
@@ -137,7 +138,7 @@ public ConfigurationAnchor recreateAnchor(ConfigurationSourceType configurationT
         }
 
         final var sources = configurationSourceRepository.findAllBySourceType(configurationType.name().toLowerCase());
-        final var now = TimeUtils.zonedDateTimeNow(ZoneId.of("UTC"));
+        final var now = TimeUtils.zonedDateTimeNow(ZoneId.of("UTC")).truncatedTo(MILLIS);
```

Comment:
seems strange as it does the change on top of util that was added because of java 17